### ISSUE TITLE
Compiling a project shows warnings for dependencies

### ISSUE THREAD

**Issue by megapctr** [gleam-lang/gleam#1643]
When building a project, I see warnings for the code in dependencies:
```
warning: Unused imported module
  ┌─ build/packages/gleam_http/src/gleam/http/response.gleam:6:8
```

[Louis said on Discord that he is aware of it and it will be fixed](https://discord.com/channels/768594524158427167/811254431051808778/977624003747414046), I'm only opening this issue for visibility.

**Comment by lpil** [comment]
This is intended behaviour, we show warnings for all code compiled.

Could you open an issue in the gleam_http library for the warnings here? Thank you

**Comment by megapctr** [comment]
I wanted to help you out with that but it turns out it was already fixed in the latest (3.0.1) version 🎉

**Comment by lpil** [comment]
Oh fab! Thank you
